packages/dev/docs/pages/releases/2024-07-22.mdx

```diff
@@ -0,0 +1,153 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+
+description: For this summer release, we have been busy working on improving virtualization in many of our Spectrum components including TableView, ListView, and ListBox. Our Color components are moving to RC, Toast focus management has been improved, we have been preparing for React 19 release along with many bug fixes and improvements.
+
+date: 2024-07-22
+---
+# July 22, 2024 Release
+
+For this summer release, we have been busy working on improving virtualization in many of our Spectrum components including TableView, ListView, and ListBox. We have significantly reduced the complexity of our implementation, resulting in smaller bundle size, better performance, and improved stability. In our testing, this resulted in 14x faster per-frame render times on large tables due to better reuse of DOM nodes as you scroll. We are also working on bringing virtualization support to React Aria Components, which is available as an unstable API in this version.
+
+Our color components are also moving to RC! Focus management within Toasts have also been greatly improved, making announcements from screen readers clearer and more consistent. Thanks to [@thmsfchtnr](https://github.com/thmsfchtnr), press events now include coordinates, enabling interactions like our new [animated ripple button example](/react-aria/examples/ripple-button.html). And finally, in preparation for React 19, this release is now compatible with React 19 RC.
+
+Thank you to our ever growing list of contributors! ❤️
+
+### API updates
+Because of various breaking changes, several packages are being bumped a major version, specifically @react-aria/virtualizer , @react-stately/virtualizer , and @react-stately/layout. While these packages were undocumented and not yet intended for public use, we know this is not always the case, so please review the updated code carefully.
```

Comment:
nit

```suggestion
Because of various breaking changes, several packages are being bumped a major version, specifically `@react-aria/virtualizer`, `@react-stately/virtualizer`, and `@react-stately/layout`. While these packages were undocumented and not yet intended for public use, we know this is not always the case, so please review the updated code carefully.
```